docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -0,0 +1,61 @@
+---
+id: build-automation
+title: Automate plugin builds
+sidebar_position: 1
+description: Automate Grafana plugin builds and releases
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - automation
+  - build
+  - automate
+  - builds
+---
+
+# Automate packaging and signing with CI
+
+If your plugin has been set up to use the supplied [Github workflows](../create-a-plugin/develop-a-plugin/set-up-github-workflows.md) from [create-plugin](../get-started/get-started.mdx),
+the plugin should be built and packaged in the correct format.
+
+If you need to set up the git worklows, see [these docs](https://github.com/grafana/plugin-actions/blob/main/build-plugin/README.md)
```

Comment:
@briangann here we should use "github workflows" not "git workflows"

Reply:
good catch, missed that one!